pallets/funding/src/functions/3_auction.rs

```diff
@@ -89,6 +89,10 @@ impl<T: Config> Pallet<T> {
 		ensure!(ct_amount > Zero::zero(), Error::<T>::TooLow);
 		ensure!(did != project_details.issuer_did, Error::<T>::ParticipationToOwnProject);
 		ensure!(matches!(project_details.status, ProjectStatus::AuctionRound), Error::<T>::IncorrectRound);
+		ensure!(
+			project_details.round_duration.started(now) && !project_details.round_duration.ended(now),
```

Comment:
Probably we can `impl` a new method in `BlockNumberPair` that is checking this, just as a syntactic sugar

Reply:
Let's keep it in mind for later. Not worth it now imo